crates/bevy_render/src/view/mod.rs

```diff
@@ -1050,10 +1050,14 @@ pub fn prepare_view_targets(
     for (entity, camera, view, texture_usage, msaa) in cameras.iter() {
         let (Some(target_size), Some(target)) = (camera.physical_target_size, &camera.target)
         else {
+            commands.entity(entity).try_remove::<ViewTarget>();
+
             continue;
         };
 
         let Some(out_attachment) = view_target_attachments.get(target) else {
+            commands.entity(entity).try_remove::<ViewTarget>();
```

Comment:
```suggestion
			// If we can't find an output attachment we need to remove the ViewTarget 
			// component to make sure the camera doesn't try rendering to an invalid 
			// output attachment.
            commands.entity(entity).try_remove::<ViewTarget>();
```

I think there should be a comment here. It took me a while to figure out _why_ this was important. The main reason is because of the retained render world.